core-services/prow/02_config/_plugins.yaml

```diff
@@ -816,6 +821,11 @@ config_updater:
         - ci
       gzip: true
       name: job-config-misc
+    ci-operator/jobs/**/*-7*.yaml:
```

Comment:
That's covering way too much, please limit the pattern up to the appropriate org or even better, a repo